docs/maintaining.md

```diff
@@ -4,11 +4,67 @@ This document is for people working on VTEX as Toolbelt maintainers.
 
 **Table of Contents**
 
+- [Deploying a new version](#deploying-a-new-version)
 - [Forcing a new version](#forcing-a-new-version)
 - [Deprecating a version](#deprecating-a-version)
 - [Changing the release message](#changing-the-release-message)
 - [Monitoring and debugging user errors](#monitoring-and-debugging-user-errors)
 
+### Deploying a new version
+
+#### **NPM**
+
+The main deploy of `toolbelt` is on [NPM](https://www.npmjs.com/package/vtex). This deploy is reponsable for warning `new and deprecated versions`
+
+Here's how you deploy on `NPM`:
+
+- Fire this command on terminal in `toolbelt` project root path (*Stable version*):
+
+```bash
+    releasy patch --stable
+    releasy minor --stable
+    releasy major --stable
+```
+
+- Fire this command on terminal in `toolbelt` project root path (*Beta version*):
+
+```bash
+    releasy patch
+    releasy minor
+    releasy major
+```
+
+This will release a `github tag` and trigger the `npm-publish github action`
+
+#### **AWS S3**
+
+This deploy will contain the `toolbelt standalone tarball` that is compatible with `MacOS` and `Linux`
+
+Here's how you deploy on `NPM`: 
+
+- Fire this command on terminal in `toolbelt` project root path
+
+```bash
+yarn release
+yarn release:win
+```
+
+This [command](https://github.com/vtex/toolbelt/blob/ef67f52cd200ab08445684767c839319b86b5454/package.json#L19) will pack and publish on AWS S3.
```

Comment:
Please talk about AWS credentials. Also mention 7-zip, that's necessary for this commands to be completed

Reply:
Awesome! I forgot that part